Flat/interface/PandaAnalyzer.h

```diff
@@ -29,6 +29,12 @@
 #include "CondFormats/JetMETObjects/interface/FactorizedJetCorrector.h"
 #include "CondFormats/JetMETObjects/interface/JetCorrectionUncertainty.h"
 
+// TMVA
+#include "TMVA/Factory.h"
```

Comment:
Could non-essential includes be removed/moved into the relevant `.cc` files? Also was this tested in 93x? I know some of the `TMVA::Factory` functionalities have changed. In fact, `PandaCore/Learning` does not work anymore and needs to be updated. I think we should use 93x has the default working environment for this release.